x-pack/plugins/observability_solution/observability_onboarding/public/assets/integrations.conf

```diff
@@ -34,9 +34,9 @@ patterns=
 title=PostgreSQL
 patterns=
     /var/log/postgresql/postgresql-*-*.log*
+    /var/log/postgresql/postgresql-*-*.csv*
```

Comment:
why is this change important?

Reply:
postres integration has it as one of the data streams, we need to detect it as well